cpp/include/cudf/replace.hpp

```diff
@@ -26,6 +26,14 @@ namespace cudf {
  * @file
  */
 
+/**
+ * @brief Policy to specify the relative position that replacement value is to the null row
```

Comment:
Clearer.

```suggestion
 * @brief Policy to specify the position of replacement values relative to null rows
```

Reply:
Lol, I literally rephrased this 2-3 times but still couldn't get it right.